### ISSUE TITLE
Version 1.2 Proposal

### ISSUE THREAD

**Issue by domenicw** [domenicw/GroupRandomizer#7]
# Version 1.2

## Proposal

Decouple people and groups. What does this mean? From now on you should be able to add as many Players (Persons) as you want. Further, assign these Players to multiple groups so you don't have to delete that person and add it afterwards again, if you don't need it for once. 
Lastly there needs to be new home screen. It should include a shortcut to the last randomized group and an option to randomize only selected players.

### Technical

For this to work, a proper backend needs to be implemented (Core Data?).
Entities: Player, Group
Also the whole UI needs to be reworked. For this new features, the old UI is not suitable. The old groups screen needs to be removed and a new groups screen needs to be added. It should contain an overview over all groups.

## TLDR

- Quick Randomize
- Add Players to multiple groups
- Player details (Name, Avatar, Description, Member of Groups)
- Group details (Name, Avatar, Description, Members)
